### PR TITLE
feat: add 'report problem' button

### PR DESCRIPTION
**Pull request by KongPobW**

### CODE REVIEW

File: src/components/core/ApplicationHeader.tsx [hunk 1]
```diff
@@ -159,6 +160,13 @@ export default function ApplicationHeader({ opened, setOpened, theme }: Applicat
 								/>
 							</div>
 						</div>
+
+						{/* Report Problem */}
+						<div className="sm:ml-0 lg:ml-3">
+							<Button variant="subtle">
+								<a className="no-underline text-sky-300" href="https://forms.office.com/r/z48ExG8dPs">Report Problem</a>
```
Comment (anascence) [inline src/components/core/ApplicationHeader.tsx]: I believe tawan asked you to add target "blank". You are missing it !

**Comment by Lxkas** [comment]
pls also use present tense for commit messages, i.e. "add report button" so that it's easier to track

**Comment by chinathaip** [comment]
#20

**Review by Lxkas — APPROVED** [review]

**Comment by chinathaip** [comment]
I've never been more happy that a PR finally gets merged

Well done @KongPobW  @Lxkas

**Comment by Lxkas** [comment]
@chinathaip 555555 why!
Now it's just on to the mobile navbar XD

**Pull request merged**